presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environments.java

```diff
@@ -47,6 +47,8 @@ public static void pruneEnvironment()
             DockerUtil.removeNetworks(
                     dockerClient,
                     listNetworksCmd -> listNetworksCmd.withNameFilter(PRODUCT_TEST_LAUNCHER_NETWORK));
+
+            DockerUtil.removeStoppedContainers(dockerClient, PRODUCT_TEST_LAUNCHER_STARTED_LABEL_NAME);
```

Comment:
The intention of `killContainers` was to stop & remove all containers, regardless runnning or stopped.
Why isn't this effective?